### PR TITLE
DCCLIP-507: Jira Grafana Dashboards

### PR DESCRIPTION
**Pull request by yzha645**
## Pull request description

This PR added Grafana dashboards for Jira. 

## Checklist
- [ ] I have added unit tests
- [ ] I have applied the change to all applicable products
- [ ] The E2E test has passed (use `e2e` label)
- [ ] (Atlassian only) Internal Bamboo CI is passing

### CODE REVIEW

File: src/test/resources/expected_helm_output/jira/output.yaml [hunk 1]
```diff
@@ -43,11 +43,19 @@ metadata:
...
-      - pattern: ".*"
+      - pattern: '(java.lang)<type=(\w+)><>(\w+):'
+        name: java_lang_$2_$3
+      - pattern: 'java.lang<type=Memory><HeapMemoryUsage>(\w+)'
+        name: java_lang_Memory_HeapMemoryUsage_$1
+      - pattern: 'java.lang<name=G1 (\w+) Generation, type=GarbageCollector><>(\w+)'
+        name: java_lang_G1_$1_Generation_$2
+      - pattern: '.*'
+    blacklistObjectNames:
+      - 'com.atlassian.jira:connectionpool=connections,connection=*,name=BasicDataSource'
```
Comment (nanux) [inline src/test/resources/expected_helm_output/jira/output.yaml:58]: Why is this hidden?

Do we want to add some comments regarding specific rules so we have the decisions and reasons documented?

File: src/test/java/test/JmxMetricsTest.java [hunk 2]
```diff
@@ -177,7 +177,7 @@ void expose_jmx_metrics_bitbucket_mesh(Product product) throws Exception {
 
         // assert jmx configmap created and has expected config
         final var jmxConfigMap = resources.getConfigMap(product.getHelmReleaseName() + "-jmx-config").getDataByKey("jmx-config.yaml");
-        assertThat(jmxConfigMap).hasTextContaining("- pattern: \".*\"");
+        assertThat(jmxConfigMap).hasTextContaining("- pattern: ");
```
Comment (nanux) [inline src/test/java/test/JmxMetricsTest.java:180]: Can we check for more?

Comment (nanux) [inline src/test/java/test/JmxMetricsTest.java:180]: Can we check for more?

**Review by nanux — APPROVED** [review]

**Comment by nanux** [comment]
Can you include some screenshots of the added dashboards?

**Comment by errcode1202** [comment]
> Can you include some screenshots of the added dashboards?

File: src/test/java/test/JmxMetricsTest.java (hunk 2 above)
Comment (bianchi2) [reply src/test/java/test/JmxMetricsTest.java:180]: We will, after merging all the products, since some of them use config from the common chart.

**Review by bianchi2 — APPROVED** [review]

**Pull request merged**